src/components/SideMenu/SideMenu.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import SideMenu from '.';
+
+describe('SideMenu', () => {
+  beforeEach(() => {
+    render(<SideMenu />);
+  });
+
+  it('renders SideMenu and its elements', () => {
+    const sideMenu = screen.getByTestId('sideMenu');
+    expect(sideMenu.children.length).toEqual(1);
+    expect(sideMenu.children[0].children.length).toEqual(2);
```

Comment:
You can enrich your tests by looking for specific components like buttons, input texts or content that can be changed based on the arguments passed to the component.

Reply:
Testing only for the number of children inside the main container at each component could be the cause of a low coverage.